src/service/config.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+package helium_config;
```

Comment:
```suggestion
package helium.config;
```
The convention we’ve followed so far is dot-separating the elements of the package name for services